docs/changelogs/CHANGELOG-2.25.md

```diff
@@ -0,0 +1,216 @@
+- Resolved an issue where logs were duplicated when multiple pods from the same service were deployed on the same Kubernetes node ([#13109](https://github.com/kubermatic/kubermatic/pull/13109))
+
+
+### Updates
+
+- Update KKP images to Alpine 3.18; auxiliary single-binary images (alertmanager-authorization-server, network-interface-manager, s3-exporter and user-ssh-keys-agent) have been changed to use `gcr.io/distroless/static-debian12` as the base image ([#12870](https://github.com/kubermatic/kubermatic/pull/12870))
+- Update metering to v1.1.2, fixing an error when a custom CA bundle is used ([#13013](https://github.com/kubermatic/kubermatic/pull/13013))
+- Update metrics-server to v0.7.0 ([#13056](https://github.com/kubermatic/kubermatic/pull/13056))
+- Update telemetry to v0.5.0, now shipping with a distroless image ([#13055](https://github.com/kubermatic/kubermatic/pull/13055))
+- Update to Go 1.22.0 ([#13057](https://github.com/kubermatic/kubermatic/pull/13057))
+- Update to Kubernetes 1.29 / controller-runtime 0.17.1 ([#13066](https://github.com/kubermatic/kubermatic/pull/13066))
+- Update Vertical Pod Autoscaler to 1.0 ([#12863](https://github.com/kubermatic/kubermatic/pull/12863))
+- Increase the default resources for VPA components to prevent OOMs ([#12887](https://github.com/kubermatic/kubermatic/pull/12887))
+
+
+### Cleanup
+
+- Remove `CloudControllerReconcilledSuccessfully` (double L) Cluster condition, which was deprecated in KKP 2.21 and has since been replaced with `CloudControllerReconciledSuccessfully` (single L) ([#12867](https://github.com/kubermatic/kubermatic/pull/12867))
+- Remove CriticalAddonsOnly toleration from node-local-dns DaemonSet as it has more general tolerations configured ([#12957](https://github.com/kubermatic/kubermatic/pull/12957))
+- Some of high cardinality metrics were dropped from User-Cluster MLA prometheus. If your KKP installation was using some of those metrics for custom Grafana dashboards for user-clusters, your dashboards might stop showing some of the charts ([#12756](https://github.com/kubermatic/kubermatic/pull/12756))
+- Deprecate v1.11 and v1.12 Cilium and Hubble KKP Addons, as Cilium CNI is managed by Applications from version 1.13 ([#12848](https://github.com/kubermatic/kubermatic/pull/12848))
+
+
+### Documentation
+
+- Examples now include command to generate secrets that works on vanilla macOS ([#12974](https://github.com/kubermatic/kubermatic/pull/12974))
+
+### Miscellaneous
+
+- Addon manifests are now loaded once upon startup of the seed-controller-manager instead of during every reconciliation. Invalid addons will now send the seed-controller-manager into a crash loop ([#12684](https://github.com/kubermatic/kubermatic/pull/12684))
+- Kube state metrics can be configured to get metrics for custom kubernetes resources ([#13027](https://github.com/kubermatic/kubermatic/pull/13027))
+
+### Dashboard & API
```

Comment:
Is this empty or should be one level higher?

Reply:
Fixed the formatting